qutebrowser/browser/hints.py

```diff
@@ -959,3 +975,112 @@ def on_mode_left(self, mode):
             # hinting.
             return
         self._cleanup()
+
+
+class WordHinter:
+
+    """Generator for word hints.
+
+    Class attributes:
+
+    Attributes:
+
+    """
```

Comment:
Either shorten the docstring so it's one line, or add docs for `words` here :wink: